### PR TITLE
feat(labware-creator): add dynamic footprint copy for tip racks

### LINKED ISSUE #7715: LC: Custom Tip Rack footprint section render conditional instruction
I’d like to see copy in the Footprint section reference Tip Racks specifically. Copy needs to be updated from existing LC sections to indicate Tip Racks, if a tip rack is selected in LC

- If user has selected custom tip rack (far left text below), render the custom tip rack instructions in the footprint section

Acceptance Criteria:

- [ ] Use "Total Footprint" as the title of the section
- [ ] New copy in description reads according to screen above
- [ ] Copy change can be confirmed through util (contains Text)

Relates to [7163 - New footprint error](https://app.zenhub.com/workspaces/opentrons-spddrs-5e1f591ca60369aa0e6b8eaf/issues/opentrons/opentrons/7163)

### PR DESCRIPTION
**Pull request by smb2268**
# Overview

Closes #7715 by adding dynamic footprint copy for tip racks

# Changelog

- Added dynamic copy to footprint for tip racks
- Converted footprint test to `react-testing-library`
- Added test coverage for dynamic tip rack copy

# Review requests

- [ ] Dynamic footprint copy appears when tip rack is selected
- [ ] Copy does not appear when any other labware type is selected

# Risk assessment

Low risk - dynamic copy in labware creator only

### CODE REVIEW

File: labware-library/src/labware-creator/components/__tests__/sections/Footprint.test.tsx [hunk 1]
```diff
@@ -84,17 +35,60 @@ describe('Footprint', () => {
...
+      screen.queryByText(
+        nestedTextMatcher(
+          'If your Tip Rack has an adapter, place it in the adapter.'
+        )
+      )
+    ).toBe(null)
+  })
+
+  it('should render form alert when error is present', () => {
+    // formikConfig.initialValues.footprintXDimension = '130'
```
Comment (IanLondon) [inline labware-library/src/labware-creator/components/__tests__/sections/Footprint.test.tsx]: leftover comment

**Review by IanLondon — APPROVED** [review]
🎉

**Review by Kadee80 — APPROVED** [review]
🎊

**Review by shlokamin — APPROVED** [review]
Woo @smb2268! 🥳

**Pull request merged**